distuv/norm_example_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv_test
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/gonum/stat"
+	"github.com/gonum/stat/distuv"
+)
+
+func Example_normal() {
+	const (
+		nmax = 1000
+		seed = 1234
+	)
+
+	// Create a normal distribution
+	dist := distuv.Normal{
+		Mu:     0,
+		Sigma:  1,
+		Source: rand.New(rand.NewSource(seed)),
```

Comment:
Can you have the main example be using `nil`? We don't want the typical user to blindly copy this code and then get the same random numbers many times. Using a non-nil source is an advanced use that should be done with care.